docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,90 @@
+# Release 372 (23 Feb 2022)
+
+## General
+
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}` 11030`)
+
+## Base-JDBC connector library
+
+* Fix spurious query failures when metadata cache is not enabled, and data
+  access depends on the session state. ({issue}`11068`)
+
+## Accumulo connector
+
+* Fix incorrect date results on `date` column type. ({issue}`11055`)
```

Comment:
```suggestion
* Fix incorrect date results when querying columns of `date` type. ({issue}`11055`)
```

Also do we have convention to refer to data-types in uppercase? (I'm not sure myself).

Reply:
no .. SQL keywords are uppercase .. for data types.. its mixed